src/foam/nanos/ruler/RuleDAO.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.ruler',
+  name: 'RuleDAO',
```

Comment:
RuleDAO suggests its a DAO for storing Rules. Maybe RULERDAO or RulerDAO would be better.